nemo_text_processing/text_normalization/en/verbalizers/cardinal.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, deterministic: bool = True):
         else:
             integer = (
                 pynini.closure(NEMO_NOT_QUOTE)
-                + pynini.closure(pynini.cross("hundred ", "") | pynini.cross("hundred ", "hundred and "), 0, 1)
+                + pynini.closure(pynini.cross("hundred ", "hundred and "), 0, 1)
```

Comment:
what is this rule for?

Reply:
`140` -> `one hundred forty` and `one hundred and forty`, just a different way of saying